### DISCUSSION TITLE
New feature to customize the practice missed words training.

### DISCUSSION THREAD

**Discussion by Krishnasai3cks** [monkeytypegame/monkeytype#1349] [Ideas]
Need a new feature to select the number of times each word repeats while doing the practice missed words training. I don't like to type the same damn word for 50 times!. If there is a feature for this then please guide me through it because I cannot seem to find it

**Comment by unrealapex** [comment]
I understand it is repetitive, but practice makes perfect.

**Reply by unrealapex** [reply]
It would be nice to have a feature that could be used to copy the incorrect word list. I'll turn this into a new discussion.

**Comment by dylan-chong** [comment]
It's only a practice test. Why not just exit the practice test?
